app/src/main/res/layout/activity_sale_information.xml

```diff
@@ -0,0 +1,213 @@
+        android:id="@+id/sale_information_value_condition"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="32dp"
+        android:layout_marginEnd="16dp"
+        android:text="@string/condition"
+        app:layout_constraintBottom_toBottomOf="@+id/sale_information_condition"
+        app:layout_constraintEnd_toStartOf="@+id/guideline50"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="@+id/sale_information_condition" />
+
+    <TextView
+        android:id="@+id/sale_information_condition"
+        android:layout_width="0dp"
+        android:layout_height="22dp"
+        android:layout_marginStart="16dp"
+        android:layout_marginEnd="32dp"
+        android:layout_marginBottom="16dp"
+        android:gravity="end"
+        app:layout_constraintBottom_toTopOf="@+id/sale_information_price"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="@+id/guideline50" />
+
+    <TextView
+        android:id="@+id/sale_information_value_price"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="32dp"
+        android:layout_marginEnd="16dp"
+        android:text="@string/price"
+        app:layout_constraintBottom_toBottomOf="@+id/sale_information_price"
+        app:layout_constraintEnd_toStartOf="@+id/guideline50"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="@+id/sale_information_price" />
+
+    <TextView
+        android:id="@+id/sale_information_price"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginEnd="16dp"
+        android:layout_marginBottom="16dp"
+        android:gravity="end"
+        app:layout_constraintBottom_toTopOf="@+id/sale_information_button_buy"
+        app:layout_constraintEnd_toStartOf="@+id/sale_information_value_currency"
+        app:layout_constraintStart_toStartOf="@+id/guideline50" />
+
+    <TextView
+        android:id="@+id/sale_information_value_currency"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="32dp"
+        android:text="@string/currency"
+        app:layout_constraintBottom_toBottomOf="@+id/sale_information_price"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="@+id/sale_information_price" />
+
+    <Button
+        android:id="@+id/sale_information_button_buy"
+        style="@style/SmallButton"
+        android:layout_marginBottom="32dp"
+        android:text="@string/value_buy"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
You should also add these informations:
WHO is selling the book (maybe the username with a link to the profile of the user) and WHERE the seller want to meet.

Reply:
Interesting point! 
But should we reveal the seller's information (or simply have to book/buy button link the buyer and seller)?